playbilling/src/main/java/com/google/androidbrowserhelper/playbilling/bundles/QuerySkuDetailsCall.kt

```diff
@@ -0,0 +1,31 @@
+package com.google.androidbrowserhelper.playbilling.bundles
+
+import android.os.Bundle
+import com.android.billingclient.api.SkuDetailsParams
+import com.android.billingclient.api.SkuDetailsResponseListener
+
+private const val CLASS_NAME = "QuerySkuDetailsCall"
+private const val PARAMS_KEY = "params"
+private const val CALLBACK_KEY = "callback"
+
+/** Holds the parameters and result callback for a call to PlayBilling#querySkuDetailsAsync. */
+internal data class QuerySkuDetailsCall(
+        val params: SkuDetailsParams,
+        val listener: SkuDetailsResponseListener) {
+
+    fun toBundle() = makeBundle(CLASS_NAME) {
+        putBundle(PARAMS_KEY, params.toBundle())
+        putBinder(CALLBACK_KEY, listener.toBinder())
+    }
+
+    companion object {
+        fun fromBundle(bundle: Bundle): QuerySkuDetailsCall {
```

Comment:
If you want to be consistent with the other files in /bundles/ you could put this on the Bundle too:
`fun Bundle.toQuerySkuDetailsCall(): QuerySkuDetailsCall { ... }`

Reply:
I'd left this one as-is at the moment because QuerySkuDetailsCall is going to be the single public class and probably called from Java and `QuerySkuDetailsCall.fromBundle(Bundle)` looks nicer than `QuerySkuDetailsCall.toQuerySkuDetailsCall(Bundle)`.

The API is fairly alpha at the moment though.

Reply:
(Got this working once I added the missing `@JvmStatic` annotation.)